SparkPost/2web2ui
# FE-1341 - Bump minor version of hibana matchbox

### What Changed
- Hibana matchbox 4.7.2 -> 4.7.3
- Release Notes: https://github.com/SparkPost/matchbox/releases/tag/v4.7.3

### How To Test
 - Check out banners for empty states, make sure the X in the top right is clickable at all screen widths.

### To Do
- [ ] Check out other release items on the release notes, and make sure it's all as expected.
